content/en/serverless/installation/ruby.md

```diff
@@ -24,8 +24,88 @@ The [Datadog Forwarder Lambda function][1] is required to ingest AWS Lambda trac
 
 ## Configuration
 
+{{< tabs >}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+You can also add the [instrumentation command](#instrumentation) to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```

Comment:
```suggestion
You can also add the [instrumentation command](#instrument) to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```